### PR TITLE
Naming FAQ format and editing

### PR DESCRIPTION
**Pull request by dpmidd**
I wanted to clarify the documentation before jumping into a publishing guide, let me know what you think.

### CODE REVIEW

**Comment by dpmidd** [comment]
@tzarebczan "Not sure I'd start out with this, confusing for average users." I was hoping my PR would clarify the document and change the tone to make it more consumable for the average user.

"This seems out of place, is it referring to the above table?" I'm not sure what this means.

"Maybe add a link somewhere to https://lbry.io/faq/claimtrie-implementation for more details about takeover/claims?" This will come separately.

**Comment by jsigwart** [comment]
The abbreviation "uri" should be capitalized "URI" consistently.

I would like to understand takeaway points 2 & 3 better. I realize there is no existing documentation on how this part works, and I was wondering how the length of the bidding process to keep control of a name is determined? As a user, I am wondering where I go to bid, how I can contribute to help someone keep a name, etc. So much unanswered about this whole process.

**Comment by dpmidd** [comment]
@jsigwart uri syntax is fixed.  I'm working on a separate publishing guide.

File: content/faq/naming.md [hunk 1]
```diff
@@ -26,29 +26,27 @@ So we thought, “What if there is a better way?” Consulting with economists,
 | **Channel** | `lbry://<@channel_name>` | A claim containing a public key, the private counterpart is used for signing other name claims. Channel claims can be specified in the uri with the `#` modifier, and the unmodified uri has vanity resolution. |
 | **Signed** | `lbry://<@channel_name>/<example>` | The claim to `<example>` in the channel of (signed by) `<@channel_name>`. The channel claim defaults to the vanity resolution, but can be specified with the `#` modifier given before the `/`.
 
-More details on how to publish to premanent, vanity, channel, and signed.
+#### What is a claim?</br>
 
-What is a claim?</br>
+Claims are vanity names will be controlled by the people who get the most value out of them. For example, Radiohead would get a lot more value out of lbry://amoonshapedpool than a squatter, pirate, or troll.
```
Comment (tzarebczan) [inline content/faq/naming.md]: "Claims are vanity names will be" ? Probably not good to start talking about vanity claims before explaining what a claim is.

**Comment by tzarebczan** [comment]
@kauff can you give this a look over and then merge?

**Comment by kauffj** [comment]
I reworked this significantly. @tzarebczan please merge after review. @dpmidd if you have further feedback please share as well - thank you for prompting this.

**Pull request merged**